apps/hash-ai-worker-ts/docker/Dockerfile

```diff
@@ -1,40 +1,57 @@
-FROM node:22.12-alpine AS base
+FROM bitnami/minideb:bookworm AS mise
+
+ENV MISE_DATA_DIR="/mise"
+ENV MISE_CACHE_DIR="/mise/cache"
+ENV MISE_INSTALL_PATH="/usr/local/bin/mise"
+ENV MISE_NODE_COREPACK=1
+ENV PATH="/mise/shims:$PATH"
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends bash curl ca-certificates && \
+    curl https://mise.run | sh && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
+COPY .config/mise /etc/mise
+
+RUN eval "$(mise activate bash)"
+
+
+FROM mise AS base
 
 WORKDIR /app
 
-COPY package.json .
-RUN apk add --no-cache jq yq && \
-    npm install -g "turbo@$(jq -r '.devDependencies.turbo' < package.json)"
+RUN mise install node && \
+    mise use --global yq npm:turbo@$(jq -r '.devDependencies.turbo' < package.json)
```

Comment:
non-blocking: I wonder if in the future we can simply have turbo installed via mise and not yarn

Reply:
Same as above: I'll add it in a follow-up.